### DISCUSSION TITLE
Laufwerk mounten - dauerhaft

### DISCUSSION THREAD

**Discussion by pekae** [OpenCCU/OpenCCU#2211] [🇩🇪 Fragen&Antworten] [answered]
ich möchte ein Laufwerk dauerhaft mounten .

ich habe in der fstab einen mount Befehl reingeschrieben ,
dieser funktioniert auch.

nach dem Reboot ist der Eintrag in der fstab aber wieder gelöscht - wieso ?
wie kann ich das verhindern

wie kann ich einen mount dauerhaft einstellen ?

mount /dev/sda3 /mnt/usbstick ist der shell Befehl

**Comment by jens-maus** [comment, marked as answer]
RaspberryMatic nutzt als zugrundeliegendes Betriebssystem ja kein Standard Ubuntu oder Debian-Linux. Es nutzt das bei embedded Systemen weit verbreitete "Buildroot" Linux. Zusätzlich fährt es zu schonung des genutzten Datenträgers (microSD karte) ein read-only rootfs das auch bei nächsten durchgeführten Update automatisch (und gewollt) komplett gelöscht wird. Der bei allgemeinen Linux-Systemen übliche Weg einfach etwas in die `/etc/fstab` einzutragen ist also keine gute Idee und wird auch so nicht funktionieren.

Um jedoch das gleiche zu erreichen muss man nur einen Blick in die offizielle Dokumentation werfen:

https://github.com/jens-maus/RaspberryMatic/wiki/Experten-Features#anbinden-einer-lokalen-partition-zb-zus%C3%A4tzliche-usb-ssd
https://github.com/jens-maus/RaspberryMatic/wiki/Experten-Features#eigene-aktionen-w%C3%A4hrend-des-bootvorgangs

Dort sollte ausreichend beschrieben werden wie man eigene Kommandos beim hochfahren der RaspberryMatic zentrale (also z.B. solch ein `mount` Befehl durchführen lassen kann.

Zu bedenken möchte ich jedoch geben, das gerade ein angesteckter USB Stick vollautomatisch gemountet werden sollte wenn dieser in einem bekannten Dateisystem (NTFS, FAT, ext4, f2fs, etc.) vorformatiert wurde. Und dann steht dieser auch automatisch für die Diagrammfunktion und das automatische nächtlich Backup zur Verfügung. Es gibt also normal keinen Grund dieses "mounten" selbstständig durchzuführen wenn es sich dabei um einen normalen USB-Stick bzw. eine externe Festplatte handelt.

**Reply by pekae** [reply]
zuvor gegooglet nix gefunden - sorry für die Mühe, funktioniert nun
